api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -1,34 +1,75 @@
 package org.pmiops.workbench.firecloud;
 
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.logging.Logger;
+import org.pmiops.workbench.api.ProfileController;
 import org.pmiops.workbench.auth.UserAuthentication;
+import org.pmiops.workbench.exceptions.ServerErrorException;
 import org.pmiops.workbench.firecloud.api.BillingApi;
 import org.pmiops.workbench.firecloud.api.ProfileApi;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ScopedProxyMode;
 import org.springframework.web.context.annotation.RequestScope;
 
 @org.springframework.context.annotation.Configuration
 public class FireCloudConfig {
 
-  @Bean
+  private static final Logger log = Logger.getLogger(FireCloudConfig.class.getName());
+
+  private static final String END_USER_API_CLIENT = "endUserApiClient";
+  private static final String ALL_OF_US_API_CLIENT = "allOfUsApiClient";
+
+  private static final String[] BILLING_SCOPES = new String[] {
+      "https://www.googleapis.com/auth/userinfo.profile",
+      "https://www.googleapis.com/auth/userinfo.email",
+      "https://www.googleapis.com/auth/cloud-billing"
+  };
+
+  @Bean(name=END_USER_API_CLIENT)
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
   public ApiClient fireCloudApiClient(UserAuthentication userAuthentication) {
     ApiClient apiClient = new ApiClient();
     apiClient.setAccessToken(userAuthentication.getCredentials());
+    // Uncomment to enable REST API debugging.
+    //apiClient.setDebugging(true);
```

Comment:
Do we think we should have a debug flag that gets turned on/off as a whole, rather than uncommenting lines individually? (Possibly as a command line or config argument?)

Reply:
A configuration setting would be fine. I'll postpone that until a later PR for config stuff.